osf/models/node.py

```diff
@@ -2351,6 +2351,9 @@ def is_fork_of(self, other):
     def is_registration_of(self, other):
         return self.is_derived_from(other, 'registered_from')
 
+    def get_links(self):
```

Comment:
`get_verified_links`, I think this is more descriptive and less ambiguous since node has many other links.

Reply:
And to be consistent, we can rename `get_links()` for `gv_translation` and `gv_requests` too.